src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -67,22 +76,38 @@ public ScanResults scan(ScanRequest scanRequest) {
             Integer projectId = cxScanParams.getProjectId();
 
             log.info("Checking if there is any existing scan for Project: {}", projectId);
-            Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);
+                Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);
+
+            String scanComment = getComment(scanRequest);
+            if(cxProperties.getEnablePostActionMonitor()) {
```

Comment:
why the detailed comment associated specifically with the 'post monitor action'?
maybe it should be a separated property (addDetailedComment)

Reply:
The comment is where we store the information we need the postback to resume the transaction. Ken and I had explored a couple of avenues including using custom fields. Essentially figure out we could package the info and store it there.

I look at older version of the code and I just added a tab to the line with 

  Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);

Its just a typo, I didn't change anything related that.